### DISCUSSION TITLE
RDP on Linux

### DISCUSSION THREAD

**Discussion by Pshemas** [Ylianst/MeshCentral#6196] [Ideas]
Currently on Linux clients I don't see an option to connect via RDP. Would it be possible to enable such option?

Here's why - Wayland is currently not supported on MC and I assume it will be a lot of work to get it done (hope I'm mistaken ;) ). So on newer distros you basically have no remote desktop via MC. 
But there's RDP on Wayland working. I can connect MC server via Wireguard so it would have access to the those Linux machines. So if I could use RDP that would be a temporary solution - and I wouldn't have to introduce additional tools to the pipeline.

**Comment by blarg3891** [comment]
Was about to request this as well.  Would love this option!!  I currently use guacamole but would much prefer mesh.
